Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -877,7 +878,8 @@ commentOrAnnotation :
 ;
 remComment : REM whiteSpace? commentBody;
 comment : SINGLEQUOTE commentBody;
-commentBody : (LINE_CONTINUATION | ~NEWLINE)*;
+// commentBody must terminate with a NEWLINE, see VBA Language spec, section 3.3.1 and 3789
+commentBody : (LINE_CONTINUATION | ~NEWLINE)* (NEWLINE | EOF);
```

Comment:
Why exactly do we explicitly allow `LINE_CONTINUATION` here. It is included in `~NEWLINE`. (It is a token and thus no subtokens are visible to the parser.)